app/presenters/specialist_document_presenter.rb

```diff
@@ -85,6 +85,10 @@ def images
     }
   end
 
+  def show_metadata_block
```

Comment:
Nitpick: I'd rename this

```suggestion
  def show_metadata_block?
```